code/Test_definitions/device-roaming-status-subscriptions.feature

```diff
@@ -0,0 +1,238 @@
+  Scenario: Receive notification for roaming-off event
+    Given that subscriptions are created synchronously
+    And a valid subscription request body  
+    And the request body property "$.type" is "roaming-off"	
+    When the request "createSubscription" is sent
+    Then the response code is 201
+    And if the device switch from roaming "ON" to roaming "OFF"
+    Then event notification "roaming-off" is received on callback-url
+    And sink credentials are received as expected
+    And notification body complies with the OAS schema at "##/components/schemas/EventRoamingOff"
+    And type="org.camaraproject.roaming-status-subscriptions.v0.roaming-off"
+
+@roaming_status_subscriptions_09_Receive_notification_when_roaming_status_changed
+  Scenario: Receive notification for roaming-status changes
+    Given that subscriptions are created synchronously
+    And a valid subscription request body  
+    And the request body property "$.type" is "roaming-status"	
+    When the request "createSubscription" is sent
+    Then the response code is 201
+    And if the device roaming-status changes
+    Then event notification "roaming-status" is received on callback-url
+    And sink credentials are received as expected
+    And notification body complies with the OAS schema at "##/components/schemas/EventRoamingStatus"
+    And type="org.camaraproject.roaming-status-subscriptions.v0.roaming-status"
+
+@roaming_status_subscriptions_10_Receive_notification_when_roaming_change_country
+  Scenario: Receive notification for roaming-change-country
+    Given that subscriptions are created synchronously
+    And a valid subscription request body  
+    And the request body property "$.type" is "roaming-change-country"	
+    When the request "createSubscription" is sent
+    Then the response code is 201
+    And if the device roaming country changes
+    Then event notification "roaming-change-country" is received on callback-url
+    And sink credentials are received as expected
+    And notification body complies with the OAS schema at "##/components/schemas/RoamingChangeCountry"
+    And type="org.camaraproject.roaming-status-subscriptions.v0.roaming-change-country"
+
+@roaming_status_subscriptions_11_subscription_expiry
+  Scenario: Receive notification for subscription-ends event on expiry  
+    Given that subscriptions are created synchronously
+    And a valid subscription request body
+    And the request body property "$.subscriptionExpireTime" is set to a value in the near future
+    When the request "createSubscription" is sent
+    Then the response code is 201 
+    And the subscription is expired
+    And event notification "subscription-ends" is received on callback-url
+    And notification body complies with the OAS schema at "##/components/schemas/EventSubscriptionEnds"
+    And type="org.camaraproject.geofencing-subscriptions.v0.subscription-ends"
```

Comment:
The type should be `org.camaraproject.roaming-status-subscriptions.v0.subscription-ends`